pkg/dockerclient/shell_test.go

```diff
@@ -186,10 +186,11 @@ func (suite *ShellClientTestSuite) TestBuildBailOnUnknownError() {
 	suite.mockedCmdRunner.
 		On("Run",
 			mock.Anything,
-			mock.MatchedBy(func(command string) bool {
+			mock.Anything,
+			mock.MatchedBy(func(vars []interface{}) bool {
+				command := vars[0].(string)
```

Comment:
what has changed?

Reply:
This - https://github.com/nuclio/nuclio/pull/3431/files#diff-a31309e92147054eae683969a63120e1a5f65e0aef26ea29baaaaa94b208aaecR1015

Following a similar lint error as above.